tests/ui-toml/float_cmp_constant_comparisons/test.rs

```diff
@@ -15,4 +15,9 @@ fn main() {
         }
         let _ = f(1.0) == f(2.0);
     }
+    {
```

Comment:
No need to put it into a nested block. However, a comment explaining that this is checking that the lint is not emitted for literal comparisons would be nice. :+1: